apps/condo/domains/common/hooks/useCheckTLSClientCert.tsx

```diff
@@ -0,0 +1,84 @@
+import getConfig from 'next/config'
+import React, { useState } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { Modal, Button, Typography } from '@open-condo/ui'
+
+const {
+    publicRuntimeConfig,
+} = getConfig()
+
+type CheckTLSClientCertConfigType = {
+    verificationUrl: string
+    installInstructionUrl: string
+}
+
+const checkTLSClientCertConfig: CheckTLSClientCertConfigType = publicRuntimeConfig.checkTLSClientCertConfig
+
+interface IUseCheckSSLClientCert {
+    loading: boolean
+    checkSSLClientCert: () => void
+    InstallTLSClientCertModal: React.FC
+}
+
+type UseCheckSSLClientCertProps = {
+    onSuccess: () => void
+}
+
+export const useCheckTLSClientCert = ({ onSuccess }: UseCheckSSLClientCertProps): IUseCheckSSLClientCert => {
```

Comment:
Splitting logic for `checkSSLClientCert` and a model window is better. You can use this hook on other pages without the model window and for example, show some text messages related to certs rather than open the modal window.